server/controllers/finance/reports/cashflow/index.js

```diff
@@ -25,7 +25,7 @@ const TEMPLATE_BY_SERVICE = './server/controllers/finance/reports/cashflow/repor
 // expose to the API
 exports.report = report;
 exports.byService = reportByService;
-
+exports.printingReport = printingReport;
```

Comment:
Is it possible to use just `reporting` as for others reports which has emailing options, because it can be useful to have the same terminology everywhere where have used this concept of exposing report on the server side.